Readme.txt

```diff
@@ -168,80 +227,75 @@ KEY ASSIGNMENTS
-  but to submit scores you must now email him at bh1scw[at]gmail.com
-
-  "Open 2019 UZ2M Morse Runner contest" Facebook Group
-  (Still active)
-      Open contest between friends in Morse Runner in two modes,
-      Single Call and Pile-Up mode. Try 10 min training and fix
-      screenshot with date and time. Please send a screenshot to
-      this group and after we check it we will publish it!
-
-  "ZS-CW Morse Runner" Facebook Group
-      In Morse Runner, Here you will screen shot your 10 min stint,
-      (ONLY 10 min not more) paste it into this Facebook Group,
-      the idea is to try beat the previous score of the last person
-      who posted their screen shot. I'm aware we are all on different
-      levels of copy speed, but that's irrelevant because you might
-      want to match their speed, and try beat their score
-
-  "CW Freak - Morse Runner" Facebook Group
-      This is a group for Morse Games fans. I thank in advance anyone
-      who wants to share their results, their advice and their
-      impressions with fun.
-
-You can view your previous score strings using the
-File -> View Score menu command.
+  
+  If you complete a full 60-minute session in the WPX Competition mode, Morse Runner
+  will generate a score string that you can post. (A replacement Hi-Score site is needed.)
+
+    The original scoring website by Alex Shovkoplyas, VE3NEA, was
+    http://www.dxatlas.com/MorseRunner/MrScore.asp
+    but it is no longer active.
+
+    Lin Quan, BG4FQD, created the scoring website, https://www.bh1scw.com/mr/score
+    but to submit scores you must now email him at bh1scw[at]gmail.com
+
+    "Open 2019 UZ2M Morse Runner contest" Facebook Group
+    (Still active)
+        Open contest between friends in Morse Runner in two modes,
+        Single Call and Pile-Up mode. Try 10 min training and fix
+        screenshot with date and time. Please send a screenshot to
+        this group and after we check it we will publish it!
+
+    "ZS-CW Morse Runner" Facebook Group
+        In Morse Runner, Here you will screen shot your 10 min stint,
+        (ONLY 10 min not more) paste it into this Facebook Group,
+        the idea is to try beat the previous score of the last person
+        who posted their screen shot. I'm aware we are all on different
+        levels of copy speed, but that's irrelevant because you might
+        want to match their speed, and try beat their score
+
+    "CW Freak - Morse Runner" Facebook Group
+        This is a group for Morse Games fans. I thank in advance anyone
+        who wants to share their results, their advice and their
+        impressions with fun.
+
+  You can view your previous score strings using the
+  File -> View Score menu command.
 
 VERSION HISTORY
 
-Version 1.81 (Oct 2022)
+CE Version 1.82 (Dec 2022)
+  - Add ARRL DX Contest (Coded by W7SST)
+  - Add Help | First Time Setup (KD4SIR & W7SST)
+  - Add cut numbers (e.g. 5NN = 599) in exchanges received
```

Comment:
oops.. I need to ask who helped code the received cut numbers.

Reply:
w7sst implemented the cut number changes.